librocksdb_sys/crocksdb/c.cc

```diff
@@ -854,6 +854,7 @@ void crocksdb_multi_get_cf(
       values_list_sizes[i] = 0;
       if (!statuses[i].IsNotFound()) {
         errs[i] = strdup(statuses[i].ToString().c_str());
+        break;
```

Comment:
`errs` should not be accessed as an array any more. Because only the first error is cared now. And copied values should be dropped.